app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for human in people:
+        person = Person(human["name"], human["age"])
+        person_list.append(person)
+
+    for human in people:
+        for person in person_list:
+            human_is_person = person.name == human["name"]
+            has_wife = "wife" in human and human["wife"] is not None
+            has_husband = "husband" in human and human["husband"] is not None
+            if human_is_person and has_wife:
+                person.wife = person.__class__.people[human["wife"]]
+            elif human_is_person and has_husband:
+                person.husband = person.__class__.people[human["husband"]]
+            else:
```

Comment:
why are you using "else: pass" here

Reply:
changed to "continue"

Reply:
construction "else: continue" is also redundant here, I advise you to simply highlight these 2 lines

Reply:
understood. done